Sources/ComposableArchitecture/Reducer/Reducers/Reduce.swift

```diff
@@ -6,12 +6,19 @@ public struct Reduce<State, Action>: ReducerProtocol {
   @usableFromInline
   let reduce: (inout State, Action) -> Effect<Action, Never>
 
+  @usableFromInline
+  static func create(
+    internal reduce: @escaping (inout State, Action) -> Effect<Action, Never>
+  ) -> Self {
+    Self(reduce)
+  }
+
```

Comment:
can't use the internal initializer trick I did in other spots because it causes ambiguity in tests due to `@testable import`. Still feel like there must be a better way...